diff --git a/src/jsonrpc/CMakeLists.txt b/src/jsonrpc/CMakeLists.txt
index 79e8515..4e93eef 100644
--- a/src/jsonrpc/CMakeLists.txt
+++ b/src/jsonrpc/CMakeLists.txt
@@ -12,7 +12,7 @@ set(VERSION_STRING ${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION})
 set_target_properties(jsonrpc jsonrpcStatic PROPERTIES VERSION "${VERSION_STRING}" SOVERSION "${VERSION_MAJOR}")
 
 
-target_link_libraries(jsonrpc ${CURL_LIBRARIES})
+target_link_libraries(jsonrpc ${CURL_LIBRARIES} dl pthread)
 target_link_libraries(jsonrpcStatic ${CURL_LIBRARIES})
 
 install(FILES ${jsonrpc_header} DESTINATION include/jsonrpc) 
