--- avfmediaplayersession.mm	2014-09-11 12:48:26.000000000 +0200
+++ avfmediaplayersessionPatch.mm	2014-12-01 12:53:14.000000000 +0100
@@ -295,7 +295,7 @@
     //AVPlayerItem "status" property value observer.
     if (context == AVFMediaPlayerSessionObserverStatusObservationContext)
     {
-        AVPlayerStatus status = [[change objectForKey:NSKeyValueChangeNewKey] integerValue];
+        AVPlayerStatus status = (AVPlayerStatus)[[change objectForKey:NSKeyValueChangeNewKey] integerValue];
         switch (status)
         {
             //Indicates that the status of the player is not yet known because
